### DISCUSSION TITLE
How to change an HTML section header font and font size?

### DISCUSSION THREAD

**Discussion by sbourdeaud** [EvotecIT/PSWriteHTML#195] [Q&A] [answered]
Don't see an option to make the font bigger for section headers.

**Comment by PrzemyslawKlys** [comment, marked as answer]
This will be available in 0.0.133

```
New-HTMLSection -HeaderText 'Test Header' -HeaderTextSize 20 {

}
```

![image](https://user-images.githubusercontent.com/15063294/107746881-dda1ab00-6d16-11eb-88e2-48d45c245ba4.png)

**Reply by sbourdeaud** [reply]
Awesome. Thank you.
